src/multi.c

```diff
@@ -342,6 +347,20 @@ void unwatchAllKeys(client *c) {
     }
 }
 
+/* iterates over the watched_keys list and
+ * look for an expired key . */
+bool isWatchedKeyExpiredExists(client *c) {
+    listIter li;
+    listNode *ln;
+    watchedKey *wk;
+    if (listLength(c->watched_keys) == 0) return false;
+    listRewind(c->watched_keys,&li);
+    while((ln = listNext(&li))) {
```

Comment:
nit: there is an extra () level that could be removed.

Reply:
I think you'd get a compiler warning à la "Did you mean `==`" for assignment within a while condition without the extra parentheses. Another way to write this would be

```C
while((ln = listNext(&li)) != NULL) {
```

Reply:
I think its better to leave it the same as in 'unwatchAllKeys' function. with the extra () level.

Reply:
styling: also missing a space after the while

Reply:
i see this exact pattern `while((ln = listNext(&li))) {` exists 67 times in redis.
the one Viktor suggested (with an additional space) exists 82 times
and one without extra parenthesis isn't used.
i understand Itay copied it as is from `unwatchAllKeys`.
considering we have so many of them, i don't mind keeping it as is, i have a slight preference for the other form, both because of the space convention, and also because it makes the extra parenthesis slightly more readable.